qemu/tests/cfg/kvm_stat.cfg

```diff
@@ -0,0 +1,11 @@
+- kvm_stat:
+    virt_test_type = qemu
+    type = kvm_stat
+    depends_pkgs = "kernel-tools"
+    kvm_stat_service_path = "/usr/lib/systemd/system/kvm_stat.service"
+    kvm_stat_interval = 0.2
+    kvm_stat_start_cmd = "systemctl start kvm_stat.service"
```

Comment:
Suggest using `restart` to ensure service be restarted after the parameter be changed

Reply:
ok, I updated it .thanks